modules/related-posts/jetpack-related-posts.php

```diff
@@ -796,19 +796,23 @@ public function get_for_post_id( $post_id, array $args ) {
 			$options['size'] = $args['size'];
 		}
 
-		if ( 0 === (int) $post_id || empty( $options['size'] ) ) {
+		if (
+			! $options['enabled']
+			|| 0 === (int) $post_id
```

Comment:
Casting `$post_id` to `int` defeats the purpose of the strict comparison operator.

Reply:
True, I'm going to leave it for a future PR since that was pre-existing.

Reply:
https://github.com/Automattic/jetpack/issues/11683